spec/feature/mission_spec.rb

```diff
@@ -0,0 +1,94 @@
+require 'rails_helper'
+Capybara.current_driver = :rack_test
+
+RSpec.describe 'Missions List', type: :feature do
+  subject { page }
+  
+  def find_mission(mission)
+    Mission.find_by(title: mission.title).id
+  end
+
+  let!(:mission) { FactoryBot.create(:mission) }
+  let!(:new_mission) { FactoryBot.create(:mission) }
+  let!(:updated_mission) { FactoryBot.create(:mission) }
+
+  before do
+    visit missions_path
+  end
+ 
+  it do
+    is_expected.to have_content('任務列表')
+    is_expected.to have_content('新增任務')
+  end
+  
+  context 'when there are missions' do
+    it 'shows the missions' do
+      is_expected.to have_content('任務列表')
+      is_expected.to have_content('新增任務')
+      is_expected.to have_content('修改任務')
+      is_expected.to have_content('刪除任務')
+      is_expected.to have_content("#{mission.title}")
+      is_expected.to have_content("#{mission.content}")
+      is_expected.to have_content("#{mission.started_at.strftime('%Y/%m/%d %H:%M')}")
+      is_expected.to have_content("#{mission.ended_at.strftime('%Y/%m/%d %H:%M')}")
```

Comment:
請將此檔案其它不必要的 string interpolation 移除；此處應該不需要。
```suggestion
      is_expected.to have_content(mission.title)
      is_expected.to have_content(mission.content)
      is_expected.to have_content(mission.started_at.strftime('%Y/%m/%d %H:%M'))
      is_expected.to have_content(mission.ended_at.strftime('%Y/%m/%d %H:%M'))
```